### PR TITLE
Docs: short section on changing the scheduler in Flax

### PR DESCRIPTION
**Pull request by pcuenca**
Originated in #2155.

### CODE REVIEW

**Comment by pcuenca** [comment]
Not sure why the docs are not building, the log says:

```
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: https://huggingface.co/api/datasets/hf-doc-build/doc-build-dev/commit/main
[16](https://github.com/huggingface/diffusers/actions/runs/4057322074/jobs/6983116315#step:11:17)
```

But they build locally in my computer. Will try again in a few hours.

File: docs/source/en/using-diffusers/schedulers.mdx [hunk 1]
```diff
@@ -260,3 +261,54 @@ image
...
+```Python
+import jax
+import numpy as np
+from flax.jax_utils import replicate
+from flax.training.common_utils import shard
+
+from diffusers import FlaxStableDiffusionPipeline, FlaxDPMSolverMultistepScheduler
+
+model_id = "runwayml/stable-diffusion-v1-5"
+scheduler, scheduler_state = FlaxDDPMScheduler.from_pretrained(
```
Comment (patil-suraj) [inline docs/source/en/using-diffusers/schedulers.mdx:278]:
```suggestion
scheduler, scheduler_state = FlaxDPMSolverMultistepScheduler.from_pretrained(
```

**Review by patil-suraj — APPROVED** [review]
Thanks a lot for adding this!

File: docs/source/en/using-diffusers/schedulers.mdx (hunk 1 above)
Comment (pcuenca) [reply docs/source/en/using-diffusers/schedulers.mdx:278]: Oops, thanks!

**Pull request merged**